python-app/test/action.yml

```diff
@@ -0,0 +1,66 @@
+name: "test"
+description: "run unittest"
+
+inputs:
+  src-path:
+    description: "path to src directory"
+    required: false
+    default: "src"
+
+runs:
+  using: "composite"
+  steps:
+    - name: clone repo
+      uses: actions/checkout@v3
```

Comment:
Any reason to use `v3` here and keep `v2` elsewhere? seems like the only diff so far is nodejs support